### PR TITLE
[docs][development-production-mode] Fix the missing colon and other updates

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Refs ENG-6504

# How

This PR
- Adds a colon (for consistency) before the command block.
- Updates/fixes other verbiage issues for consistency across docs

# Test Plan

Run docs locally via yarn dev and then open http://localhost:3002/workflow/development-mode/

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/workflow/development-mode.mdx [hunk 1]
```diff
@@ -6,19 +6,21 @@ sidebar_title: Development and production
 import Video from '~/components/plugins/Video';
 import { Terminal } from '~/ui/components/Snippet';
 
-Your project will always run in one of two modes: development or production. By default, running your project locally with `expo start` runs it in development mode, whereas a published project (via `eas update`), or any standalone apps, will run in production mode.
+Your project will always run in two modes: **development** or **production**. By default, running your project locally with `npx expo start` runs it in development, whereas a published project (with `eas update`), or any standalone app, will run in production.
```
Comment (kbrandwijk) [inline docs/pages/workflow/development-mode.mdx:9]: > Your project will always run in two modes
This sounds weird to me.

File: docs/pages/workflow/development-mode.mdx [hunk 2]
```diff
@@ -6,19 +6,21 @@ sidebar_title: Development and production
...
 
-Production mode [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Development mode includes useful warnings and gives you access to tools that make development and debugging easier. Let's look at each of these modes more in detail and learn how you can toggle between them.
+**Development mode** includes useful warnings and gives you access to tools that make development and debugging easier. **Production mode** [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Let's look at each of these modes more in detail and learn how you can toggle between them.
```
Comment (kbrandwijk) [inline docs/pages/workflow/development-mode.mdx:11]: You didn't change this, but I wonder why we link out to a commercial 3rd party here.

Comment (kbrandwijk) [inline docs/pages/workflow/development-mode.mdx:11]:
```suggestion
**Development mode** includes useful warnings and gives you access to tools that make development and debugging easier. **Production mode** [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Let's look at each of these modes in more detail and learn how you can toggle between them.
```

File: docs/pages/workflow/development-mode.mdx [hunk 3]
```diff
@@ -6,19 +6,21 @@ sidebar_title: Development and production
...
+
+Development mode also performs validations while your app is running to give you warnings if, for example, you're using a deprecated property or if you forgot to pass a required property into a component. The video below shows the Element Inspector and Performance Monitor in action on both Android and iOS simulators:
 
 <Video file="dev-prod/devMode.mp4" />
 
 > **This comes at a cost: your app runs slower in development mode.**
 >
-> You can toggle it on and off with the Expo CLI, see [production mode](#production-mode). When you switch it, just close and re-open your app for the change to take effect. **Any time you are testing the performance of your app, be sure to disable development mode**.
+> You can toggle it on and off with the Expo CLI, see [Production mode](#production-mode). When you switch it, just close and re-open your app for the change to take effect. **Any time you are testing your app's performance, be sure to disable development mode**.
```
Comment (kbrandwijk) [inline docs/pages/workflow/development-mode.mdx:23]: 'toggle' _or_ 'switch'

File: docs/pages/workflow/development-mode.mdx (hunk 2 above)
Comment (amandeepmittal) [reply docs/pages/workflow/development-mode.mdx:11]: No clue, but I'd replace that link with a little description minification in our docs: https://docs.expo.dev/guides/customizing-metro/#minification

File: docs/pages/workflow/development-mode.mdx (hunk 3 above)
Comment (amandeepmittal) [reply docs/pages/workflow/development-mode.mdx:23]: I'll rephrase this part. Found another issue.

**Review by kbrandwijk — APPROVED** [review]

**Pull request merged**